docs/Level.md

```diff
@@ -249,6 +249,17 @@ ___
 ### Get·Stage·Type () {: aria-label='Functions' }
 [ ](#){: .abrep .tooltip .badge }
 #### [StageType](enums/StageType.md) GetStageType ( ) {: .copyable aria-label='Functions' }
+The `StageType` returned by this function corresponds to the letter (if any) appended to the stage number in the `stage` [console command](tutorials/DebugConsole.md#stage). For example, using this function in Stage 1 will return one of the following depending on which version of the stage the player is in:
```

Comment:
In my opinion, saying that the value returned by this function corresponds to a console command can cause confusions. Right now it sounds like the value only changes if you use the correct letter and the console command stage, which is not the case at all.

I would rather highlight that it corresponds to the variant of a stage. For example, basement is the base version of stage 1, so it gets stagetype 0, cellar is variant 1 so it gets  type 1, burning basement is 2 etc. 

Then as a second paragraph you can say, something like "using the console command 'stage', the stage Type corresponds to a letter in the alphabet. Basement would be stage 1 because it's the base variant.  cellar would be stage 1a, and burning basement 1b...

Reply:
i think it is better to just remove any mention of console commands entirely